pkg/utils/utils.go

```diff
@@ -114,3 +121,70 @@ func Int32Ptr(x int32) *int32 {
 func AddDryRunPrefix(msg string) string {
 	return fmt.Sprintf("[DRY RUN] %s", msg)
 }
+
+func UploadFile(url, filePath string, formFields map[string]any) error {
+	var buffer bytes.Buffer
+	writer := multipart.NewWriter(&buffer)
```

Comment:
We can avoid needing to reference `buffer` every time we pass it
```suggestion
	buffer := new(bytes.Buffer)
	writer := multipart.NewWriter(buffer)
```